erpnext_documentation/www/docs/user/manual/en/introduction/index.md

```diff
@@ -1,38 +1,35 @@
 <!-- add-breadcrumbs -->
 # Introduction
 
-<p class='lead'>What is an ERP system? And why should businesses use it?</p>
+## What is an ERP?
 
-> If you are already convinced you need an all-in-one enterprise solutions system for your organization, you can skip this page.
+> If you are already convinced that you need an all-in-one software for your organization, you can skip this page.
 
-Today, small and medium-sized businesses (SMEs) also face some of the problems that large enterprises do. One of them being planning and managing their resources. SMEs face similar complexities but have limited resources to deal with them.
+ERP stands for Enterprise Resource Planning. ERP is a category of business management software used to collect, store, manage, interpret data from business activities, enforce processes, enable control mechanisms and automate a range of activities.
 
-Small businesses (like their bigger counterparts) have to:
+Here is a bit of historical context on how the term ERP came to be. Manufacturing companies need to plan and manage their resources like raw materials, machinery, and workforce to meet customer demands. In the 1960s and 70s companies started to use software to better plan and manage the resources. These software were called **Material Requirements Planning (MRP or MRP I)**. From this **Manufacturing Resource Planning (MRP or MRP II)** evolved which added more functionalities like forecasting, maintenance scheduling, and financial accounting. Later more functionalities like human resource management, project planning were added by the MRP vendors in an attempt to offer an all-in-one software, and such software were named **Enterprise Resource Planning (ERP)**. While ERPs have their root in the manufacturing industry, today companies in all sorts of industries use them.
 
-- Do accounts to track their sales/purchases.
-- Do their taxes.
-- Pay their employees.
-- Manage deliveries within promised timelines.
-- Deliver quality goods and services.
-- Communicate with customers, answer their queries.
+Whether you are a tiny, small, mid, large, huge, or humongous organization, you will benefit from having an ERP.
```

Comment:
```suggestion
Whether you are a small, medium or large organization, you will benefit from having an ERP.
```

Not sure if tiny and humongous are standard company sizes. 😄 And an ERP system might be overkill for a tiny organization.